src/arch/arm/64/object/objecttype.c

```diff
@@ -495,6 +501,10 @@ exception_t Arch_decodeInvocation(word_t label, word_t length, cptr_t cptr,
     case cap_cb_cap:
         return decodeARMCBInvocation(label, length, cptr, slot, cap, call, buffer);
 #endif /*CONFIG_ARM_SMMU*/
+#ifdef CONFIG_ALLOW_SMC_CALLS
```

Comment:
I think we have to add this in line 488 also

Reply:
CONFIG_ALLOW_SMC_CALLS has a dependency on CONFIG_ARM_HYPERVISOR_SUPPORT, so it works how it is. However, we can be more explicit and add it into 488 if you would like.

Reply:
I think having this explicitly improves maintainability. How strong us the dependency of the SMC feature on the Hypervisor feature - could they be separated one day?

Reply:
We have actually already seen a use case to uncouple this from virtualization support, so we updated the specified ifdef line and changed the config dependency.